roles/variables/vars/main.yml

```diff
@@ -1852,7 +1873,8 @@ control_center_properties:
     enabled: "{{ ksql_cluster_ansible_group_names | default(omit) | length > 0 }}"
     properties: "{{ ksql_cluster_ansible_group_names | confluent.platform.c3_ksql_properties(groups, hostvars, ksql_ssl_enabled,
         ksql_http_protocol, ksql_listener_port, control_center_truststore_path, control_center_truststore_storepass,
-        control_center_keystore_path, control_center_keystore_storepass, control_center_keystore_keypass) }}"
+        control_center_keystore_path, control_center_keystore_storepass, control_center_keystore_keypass, ksql_oauth_enabled, rbac_enabled,
+        ksql_oauth_user, ksql_oauth_password, oauth_groups_scope, idp_self_signed) }}"
```

Comment:
these should be control creds. this is for c3 (client) talking to ksql (server).